### PR TITLE
Feature/update django 2.0

### PR DESCRIPTION
**Pull request by todd-dembrey**
~The only thing i have found broken in the styling of filter elements on the submissions page. Will look into that this week.~

### CODE REVIEW

File: opentech/apply/users/migrations/0004_django2_update.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.2 on 2018-03-01 21:46
```
Comment (zerolab) [inline opentech/apply/users/migrations/0004_django2_update.py]: This migration looks out of place.

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,11 @@
-Django==1.11.8
+Django==2.0.2
+djangorestframework==3.7.4
```
Comment (zerolab) [inline requirements.txt]: are we using it yet?

Comment (todd-dembrey) [reply requirements.txt]: No, but wagtail allows a broad range of versions (`djangorestframework>=3.1.3,<4.0`). This pins it to the django 2.0 compat version.
http://www.django-rest-framework.org/topics/release-notes/#374 

Might be an issue to raise with wagtail core.

File: opentech/apply/users/migrations/0004_django2_update.py (hunk 1 above)
Comment (todd-dembrey) [reply opentech/apply/users/migrations/0004_django2_update.py]: You're right...

File: requirements.txt (hunk 2 above)
Comment (zerolab) [reply requirements.txt]: I think it is worth it. I sense a minor release coming in not too far of a future containing all the little tweaks like that

**Comment by zerolab** [comment]
Other than the rogue migration, LGTM

**Pull request merged**